main.js

```diff
@@ -1,10 +1,66 @@
 window.onload = () => {
-    let menuContainer = document.getElementById("mob-container");
-    let menu = document.querySelectorAll(".mob-click");
-    menu.forEach(el => {
-        el.addEventListener('click', () => {
-            menuContainer.classList.toggle('disapear');
-        })
+  const menuContainer = document.getElementById('mob-container');
+  const menu = document.querySelectorAll('.mob-click');
+  menu.forEach((el) => {
+    el.addEventListener('click', () => {
+      menuContainer.classList.toggle('disapear');
+    });
+  });
 
-    })
-}
+  const speakersList = [
+    {
+      name: 'Bizimungu Pascal',
+      title: 'Scratch programming trainer and web developer',
+      image: './img/pascal.jpeg',
+      shortDetails: 'Bacon ipsum dolor amet prosciutto ball tip swine, shank andouille ham hock short loin',
+    },
+    {
+      name: 'Bizimungu Pascal',
+      title: 'Scratch programming trainer and web developer',
+      image: './img/speaker6.jpg',
+      shortDetails: 'Bacon ipsum dolor amet prosciutto ball tip swine, shank andouille ham hock short loin',
+    },
+    {
+      name: 'Bizimungu Pascal',
+      title: 'Scratch programming trainer and web developer',
+      image: './img/speaker3.jpg',
+      shortDetails: 'Bacon ipsum dolor amet prosciutto ball tip swine, shank andouille ham hock short loin',
+    },
+    {
+      name: 'Bizimungu Pascal',
+      title: 'Scratch programming trainer and web developer',
+      image: './img/speaker2.jpg',
+      shortDetails: 'Bacon ipsum dolor amet prosciutto ball tip swine, shank andouille ham hock short loin',
+    },
+    {
+      name: 'Bizimungu Pascal',
+      title: 'Scratch programming trainer and web developer',
+      image: './img/speacher1.jpg',
+      shortDetails: 'Bacon ipsum dolor amet prosciutto ball tip swine, shank andouille ham hock short loin',
+    },
+    {
+      name: 'Bizimungu Pascal',
+      title: 'Scratch programming trainer and web developer',
+      image: './img/pascal.jpeg',
+      shortDetails: 'Bacon ipsum dolor amet prosciutto ball tip swine, shank andouille ham hock short loin',
+    },
+  ];
+
+  const getSpeakers = document.getElementById('speaker-container');
+  getSpeakers.innerHTML = speakersList.map((el) => `
```

Comment:
Your code iterates through the speakers list and creates HTML elements with the data from each object 🔥 Great work!